ocs_ci/ocs/node.py

```diff
@@ -1481,9 +1475,12 @@ def wait_for_new_osd_node(old_osd_node_names, timeout=600):
             Else it returns None
 
     """
+    pod.wait_for_pods_to_be_running(
+        pod_names=[osd_pod.name for osd_pod in pod.get_osd_pods()], timeout=timeout
+    )
```

Comment:
I am not sure about this change... If the osd pods are running, are we sure that the new osd node should be running?

Reply:
Yes, I see. But maybe the time for the pods to reach the running state is less than the new osd node to appear in the new osd nodes. In this case, we shouldn't decrease the timeout in the next function from `timeout` to `60`.

Reply:
ack, will increase the timeout in next function same as earlier, so as to not fail in the above scenario.